### PR TITLE
Data sphinx bafu gewiss

### PR DESCRIPTION
**Pull request by AFoletti**
merged the last GEWISS batch. Thanks @pfanguin and @daguer !

### CODE REVIEW

**Comment by AFoletti** [comment]
@ltclm , could you please review and merge? Thanks!

File: conf/bafu.conf.part [hunk 1]
```diff
@@ -132,6 +132,74 @@ source src_ch_bafu_hydrologie-untersuchungsgebiete_stationen : def_searchable_fe
...
 }
 
+source src_ch_bafu_hydrologie-daueruntersuchung_fliessgewaesser : def_searchable_features
+{
+    sql_db = bafu_dev
+    sql_query = \
+        SELECT objectid as id \
+            ,  name as label \
+            , 'feature' as origin \
+            ,  remove_accents(name)||' '||remove_accents(betriebsbeginn)||' '||remove_accents(stationierung)||' '||remove_accents(flussgebiet)||' '||remove_accents(einzugsgebietsflaeche)||' '||remove_accents(mittlerehoehe)||' '||remove_accents(hoehe)||' '||remove_accents(vergletscherung) as detail \
```
Comment (ltclm) [inline conf/bafu.conf.part]: concatenated attributes have to be wrapped inside coalesce function (as described in wiki)
so please replae
```sql
remove_accents(name)||' '||...
-- with
remove_accents(coalesce(name,' '))||' '||..
```

File: conf/bafu.conf.part [hunk 2]
```diff
@@ -132,6 +132,74 @@ source src_ch_bafu_hydrologie-untersuchungsgebiete_stationen : def_searchable_fe
...
+}
+
+source src_ch_bafu_hydrologie-hochwassergrenzwertpegel : def_searchable_features
+{
+    sql_db = bafu_dev
+    sql_query = \
+        SELECT objectid as id \
+            ,  name as label \
+            , 'feature' as origin \
+            ,  remove_accents(name)||' '||hoehe::text||' '||remove_accents(fluss)||' '||einzugsgebietsflaeche::text||' '||remove_accents(m_beginn)||' '||remove_accents(m_ende) as detail \
```
Comment (ltclm) [inline conf/bafu.conf.part]: wrap attributes in coaleasce (xx,' ')

File: conf/bafu.conf.part [hunk 3]
```diff
@@ -132,6 +132,74 @@ source src_ch_bafu_hydrologie-untersuchungsgebiete_stationen : def_searchable_fe
...
+}
+
+source src_ch_bafu_hydrologie-untersuchungsgebiete : def_searchable_features
+{
+    sql_db = bafu_dev
+    sql_query = \
+        SELECT geodb_oid as id \
+            ,  name as label \
+            , 'feature' as origin \
+            , geodb_oid::text||' '||remove_accents(name)||' '||max_hoe::text||' '||min_hoe::text||''||mit_hoe::text||' '||remove_accents(station)||' '||remove_accents(regimetyp)||' '||df::text||' '||sc::text||' '||ms::text||' '||mp::text||' '||antws_tot::text||' '||antogr::text||' '||antwiack::text||' '||antweid::text||' '||antunpr::text||' '||antgeb::text||' '||antindu::text||' '||antgew_ms::text||' '||antveg_los_ov::text||' '||antv_ab86::text||' '||shape_area::text as detail \
```
Comment (ltclm) [inline conf/bafu.conf.part]: wrap attributes in coaleasce (xx,' ')

File: conf/bafu.conf.part [hunk 4]
```diff
@@ -132,6 +132,74 @@ source src_ch_bafu_hydrologie-untersuchungsgebiete_stationen : def_searchable_fe
...
+}
+
+source src_ch_bafu_hydrologischer-atlas_kantonale-messstationen : def_searchable_features
+{
+    sql_db = bafu_dev
+    sql_query = \
+        SELECT objectid as id \
+            ,  nummer as label \
+            , 'feature' as origin \
+            ,  remove_accents(nummer)||' '||betriebsbeginn::text||' '||hoehe::text||' '||einzugsgebietsflaeche::text||' '||bilanzgebietsnummer::text||' '||vergletscherung::text as detail \
```
Comment (ltclm) [inline conf/bafu.conf.part]: wrap attributes in coaleasce (xx,' ')

File: conf/bafu.conf.part [hunk 5]
```diff
@@ -387,6 +455,57 @@ source src_ch_bafu_waldreservate : def_searchable_features
...
 }
 
+source src_ch_bafu_mittlere-abfluesse : def_searchable_features
+{
+    sql_db = bafu_dev
+    sql_query = \
+        SELECT bgdi_id::bigint as id \
+            , regimenummer as label \
+            , 'feature' as origin \
+            , remove_accents(coalesce(regimetyp,' '))||' '||coalesce(regimenummer::text,' ')||' '||coalesce(abflussvar::text,' ')||' '||coalesce(mqn_jahr::text,' ')||' '||coalesce(mqn_jan::text,' ')||' '||coalesce(mqn_feb::text,' ')||' '||coalesce(mqn_mar::text,' ')||' '||coalesce(mqn_apr::text,' ')||' '||coalesce(mqn_mai::text,' ')||' '||coalesce(mqn_jun::text,' ')||' '||coalesce(mqn_jul::text,' ')||' '||coalesce(mqn_aug::text,' ')||' '||coalesce(mqn_sep::text,' ')||' '||coalesce(mqn_okt::text,' ')||' '||coalesce(mqn_nov::text,' ')||' '||coalesce(mqn_dez::text,' ') as detail \
```
Comment (ltclm) [inline conf/bafu.conf.part]: wrap attributes in coaleasce (xx,' ')

File: conf/bafu.conf.part [hunk 6]
```diff
@@ -387,6 +455,57 @@ source src_ch_bafu_waldreservate : def_searchable_features
...
+}
+
+source src_ch_bafu_wasserbau-querprofilmarken : def_searchable_features
+{
+    sql_db = bafu_dev
+    sql_query = \
+        SELECT bgdi_id::bigint as id \
+            , schluesselid as label \
+            , 'feature' as origin \
+            , remove_accents(coalesce(typ,' ')||' '||coalesce(herkunft,' ')) as detail \
```
Comment (ltclm) [inline conf/bafu.conf.part]: wrap attributes in coaleasce (xx,' ')

File: conf/bafu.conf.part [hunk 7]
```diff
@@ -387,6 +455,57 @@ source src_ch_bafu_waldreservate : def_searchable_features
...
+}
+
+source src_ch_bafu_feststoffe-geschiebemessnetz : def_searchable_features
+{
+    sql_db = bafu_dev
+    sql_query = \
+        SELECT bgdi_id::bigint as id \
+            , gsch_n as label \
+            , 'feature' as origin \
+            , remove_accents(coalesce(geologie,' ')||' '||coalesce(platz,' ')||' '||coalesce(fluss,' ')||' '||coalesce(station,' ')||' '||coalesce(institut,' ')||' '||coalesce(amt,' '))||' '||coalesce(gsch_n::text,' ')||' '||coalesce(lk::text,' ')||' '||coalesce(lage,' ')||' '||coalesce(fn::text,' ')||' '||coalesce(hmax::text,' ')||' '||coalesce(hmin::text,' ')||' '||coalesce(hmed::text,' ')||' '||coalesce(exp,' ')||' '||coalesce(form::text,' ') as detail \
```
Comment (ltclm) [inline conf/bafu.conf.part]: wrap attributes in coaleasce (xx,' ')

**Comment by ltclm** [comment]
besides the missing coalesce looks good to me.

File: conf/bafu.conf.part (hunk 6 above)
Comment (AFoletti) [inline conf/bafu.conf.part]: coalesce already present

File: conf/bafu.conf.part (hunk 7 above)
Comment (AFoletti) [inline conf/bafu.conf.part]: coalesce already present

**Comment by AFoletti** [comment]
Thanks for the review. I'll let you test again and merge if ok. :)

**Pull request merged**